entity_gym/entity_gym/environment.py

```diff
@@ -314,6 +416,9 @@ def reset(self, obs_filter: ObsSpace) -> Observation:
     def act(self, action: Mapping[str, Action], obs_filter: ObsSpace) -> Observation:
         return self.__class__.filter_obs(self._act(action), obs_filter)
 
+    def close(self) -> None:
```

Comment:
This should probably just do nothing by default rather than error, or be an `@abstractmethod`.